implementations/c/provekit-lsp-c/tests/integration.sh

```
@@ -0,0 +1,93 @@
+    echo "      Run 'make' first." >&2
+    exit 1
+fi
+
+FIXTURE="$SCRIPT_DIR/fixtures/two_funcs.c"
+SOURCE=$(cat "$FIXTURE" | sed 's/\\/\\\\/g; s/"/\\"/g; s/$/\\n/' | tr -d '\n' | sed 's/\\n$//')
+
+PASS=0
+FAIL=0
+
+check() {
+    local label="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)